README.md

```diff
@@ -33,7 +33,7 @@ The Network Analyzer (NA) Plugin is an OpenTAP plugin designed for seamless cont
 
 ### Installation:
 
-Download the NA Plugin installer from [here](https://packages.opentap.io/)
+Download the NA Plugin installer from [here](https://packages.opentap.io/#name=%2FPackages%2FNetwork+Analyzer&version=0.5.14%2B41b8f68e&os=Windows%2CLinux&architecture=AnyCPU)
```

Comment:
Change URL to this more generic form so its always referencing the latest version,  and not tied to that specific version:
https://packages.opentap.io/#name=%2FPackages%2FNetwork+Analyzer